test/org/sagebionetworks/bridge/services/SendEmailIntegTest.java

```diff
@@ -38,7 +38,7 @@ public void test() {
         final ConsentSignature signature = ConsentSignature.create("Eggplant McTester", "1970-05-01", IMG, "image/png");
         final User user = new User();
         final Study study = studyService.getStudy(TestConstants.TEST_STUDY_IDENTIFIER);
-        user.setEmail(study.getSupportEmail());
+        user.setEmail("eggplant.mctester@tester.com");
```

Comment:
This won't work either. It looks like this integration test sends email for real, and eggplant.mctester@tester.com doesn't exist. (As evidenced by the bounce notification I just received.)

You should change this to bridge-testing@sagebase.org (with or without a plus), which was created specifically for this purpose.